File: 1-iniciante/1009/1009.c
#include <stdio.h>

int main() {
    char name[30];
    double salary, sales;
    gets(name);
    scanf("%lf%lf", &salary, &sales);
    printf("TOTAL = R$ %.2lf\n", salary + (sales * 0.15));
    return 0;
}

File: 1-iniciante/1009/1009.cs
using System;

class URI {
    static void Main(string[] args) {
        string name;
        double salary, sales;
        name = Console.ReadLine();
        salary = double.Parse(Console.ReadLine());
        sales = double.Parse(Console.ReadLine());
        Console.WriteLine($"TOTAL = R$ {salary + (sales * 0.15):0.00}");
    }
}